test/git.js

```diff
@@ -1,3 +1,6 @@
+// `npm test --ignore-scripts` overrides any local options, so we need to remove it from the env
+delete process.env.npm_config_ignore_scripts
```

Comment:
tap tests run in their own environments.  This solution will likely not work if we ever switch from tap. Just something to put in the back of your head (i.e. we'll want to store/restore this instead of a blanket removal)

Reply:
makes sense!